frontend/packages/ceph-storage-plugin/src/components/disk-inventory/ocs-disks-list.tsx

```diff
@@ -193,11 +193,13 @@ const OCSDisksList: React.FC<TableProps> = React.memo((props) => {
         const { devicePath, deviceID, deviceOsd, deviceNode, deviceSerial } =
           getAnnotations(ti) || {};
         if (devicePath && deviceOsd && deviceNode === nodeName) {
-          data[devicePath] = {
+          data[ti.metadata.name] = {
```

Comment:
When the same name disk is replaced then we can run into bugs where the same disk is updated.
Hence using template name which will be different.